docs/release-notes/self-hosted/self-hosted-v3.5.0.md

```diff
@@ -0,0 +1,70 @@
+# Self-hosted v3.5.0
+
+These release notes are for [Codacy Self-hosted v3.5.0](https://github.com/codacy/chart/releases/tag/3.5.0){: target="_blank"}, released on March 31, 2021.
+
+To upgrade Codacy, follow [these instructions](../../chart/maintenance/upgrade.md).
+
+## Product enhancements
+
+-   Now, Codacy allows [configuring who is allowed to manage patterns and ignore files](https://docs.codacy.com/organizations/roles-and-permissions-for-synced-organizations/#change-analysis-configuration) on all supported Git providers. (CY-3883)
```

Comment:
I think the ignore issues to all providers is also being released now. @pedrocodacy @carlospinho04 @franciscodua @josemiguelmelo

Reply:
I think that is not yet included on this release from my understanding, but yeah, if you guys can confirm, would be great! :D

Reply:
Initially we didnt plan to release this on 3.4 but we ended up releasing it. There was a bit of a confusion with that release's notes but they have been patched to introduce this.

tldr: it was released in the previous version

Reply:
Oh ok, thx! I was a bit off about what it was then 😅  either way, we good here then, nice!